docs/index.html

```diff
@@ -8,9 +7,14 @@
 -->
 
 <!--Let's get meta bois-->
+<!-- Lots of kids play FNF, keep everything visible child friendly. -->
```

Comment:
very contradictory to what the game actually has

Reply:
the whole point of the game is to be a love letter to the mature site Newgrounds.

Reply:
> the whole point of the game is to be a love letter to the mature site Newgrounds.

cool but this was a year ago